src/runtime/auto-prompt-manager-test.js

```diff
@@ -1076,6 +1083,41 @@ describes.realWin('AutoPromptManager', {}, (env) => {
     expect(alternatePromptSpy).to.not.be.called;
   });
 
+  it('should force display the large prompt if the viewport is wider than 480px and DISABLE_DESKTOP_MINIPROMPT is enabled', async () => {
+    win./*OK*/ innerWidth = 500;
+    setExperiment(win, ExperimentFlags.DISABLE_DESKTOP_MINIPROMPT, true);
+
+    await autoPromptManager.showAutoPrompt({
+      autoPromptType: AutoPromptType.CONTRIBUTION,
+      alwaysShow: true,
+      displayLargePromptFn: alternatePromptSpy,
+    });
+    expect(alternatePromptSpy).to.be.calledOnce;
+  });
+
+  it('should only display the mini prompt if the viewport is narrower than 480px and DISABLE_DESKTOP_MINIPROMPT is enabled', async () => {
```

Comment:
The condition "viewport narrower than 480px" should be set explicit in the test.